projects/packages/jetpack-mu-wpcom/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.35.0] - 2024-06-10
+### Added
+- Jetpack-mu-wpcom: Add hosting > overview menu option on wp-admin sidebar [#37732]
+
+### Changed
+- WPCOM Block Editor: Moved from Jetpack Plugin to mu-wpcom [#37324]
+
+### Removed
+- Jetpack-mu-wpcom: Remove the hosting menu options monitoring and configuration [#37736]
+
+### Fixed
+- mu-wpcom: Fixed typo in setcookie call [#37774]
```

Comment:
Tiny tweaks. Note that the last item was a regression and could be combined with the "WPCOM Block Editor" entry.
```suggestion
- Add Hosting > Overview menu option on WP Admin sidebar. [#37732]

### Changed
- WPCOM Block Editor: Moved from Jetpack Plugin to mu-wpcom [#37324]

### Removed
- Remove Monitoring and Configuration menu options from Hosting menu. [#37736]

### Fixed
- Fixed typo in setcookie call [#37774]
```